### PR TITLE
docs(raddix): useToggle hook

### PR DESCRIPTION
**Pull request by immois**
Add documentation of useToggle hook

### CODE REVIEW

File: apps/raddix/data/docs/utilities/use-toggle/use-toggle.es.mdx [hunk 1]
```diff
@@ -1,7 +1,73 @@
...
+
+En este ejemplo, alternaremos entre mostrar más/mostrar menos texto.
+
+<CodeBlock folder='use-toggle' file='example1.tsx' />
+
+## API
+
+## Resultado
+
+El hook `useToggle` devuelve una matriz con los siguientes elementos:
```
Comment (realDiegoR) [inline apps/raddix/data/docs/utilities/use-toggle/use-toggle.es.mdx:30]: Better change "matriz" for ''arreglo" or "array"

File: apps/raddix/data/docs/utilities/use-toggle/use-toggle.es.mdx [hunk 2]
```diff
@@ -1,7 +1,73 @@
...
+
+<ApiTable
+  head={{
+    name: 'Nombre',
+    description: 'Descripción',
+    required: 'Requerido'
+  }}
+  data={[
+    {
+      name: 'valor inicial',
```
Comment (realDiegoR) [inline apps/raddix/data/docs/utilities/use-toggle/use-toggle.es.mdx:67]: Change "valor inicial" for "initialValue" as it is the param's name. It needs the default value too.

**Review by realDiegoR — COMMENTED** [review]
I like it. It needs some adjustments and it's ready to go.

**Comment by immois** [comment]
Thanks for this very thorough review @realDiegoR

**Pull request merged**